src/app.rs

```diff
@@ -307,11 +307,18 @@ impl<'a, 'b, T> ApplicationBuilder<'a, 'b, T> {
 
     pub fn new<P: AsRef<Path>>(path: P, initial_state: T) -> Result<Self> {
         use bundle::AppBundle;
+        use rustc_version;
 
         println!("Initializing Amethyst...");
         println!("Version: {}", vergen::semver());
         println!("Platform: {}", vergen::target());
-        println!("Git commit: {}", vergen::sha());
+        println!("Amethyst git commit: {}", vergen::sha());
+        if let Ok(rustc_meta) = rustc_version::version_meta() {
+            println!("Rustc version: {} {:?}", rustc_meta.semver, rustc_meta.channel);
+            if let Some(hash) = rustc_meta.commit_hash {
+                println!("Rustc git commit: {}", hash);
+            }
+        }
```

Comment:
Under which conditions can `rustc_version::version_meta()` fail? Would it make sense to write the error on `Err`? Something like

> Rustc version: Error: unknown pre-release tag

Reply:
This will fail if it wasn't able to parse the version output from rustc during the build process.  It seems unlikely this would happen so I didn't code anything for it.  If I turn out to be wrong I'll add a failure check here.